Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionControllerDelegatePrivate.h

```diff
@@ -67,6 +69,8 @@ WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA))
  */
 - (void)_webExtensionController:(WKWebExtensionController *)controller didCreateBackgroundWebView:(WKWebView *)webView forExtensionContext:(WKWebExtensionContext *)context;
 
+- (void)_webExtensionController:(WKWebExtensionController *)controller presentSidebar:(_WKWebExtensionSidebar *)sidebar forExtensionContext:(WKWebExtensionContext *)context completionHandler:(void (^)(NSError * _Nullable error))completionHandler;
```

Comment:
Should add header doc here too so it is ready to go.

Reply:
Whoops, missed this one before I opened the PR -- adding now